### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by plafrance**
I reverted commit #872fcd5 because I was unable to reproduce the bug. Also, I fixed a bug that crashed the browser when trying to look for completions when the network in unavailable.

### CODE REVIEW

**Comment by parkouss** [comment]
Thanks for looking at it again - it seems that indeed I can not reproduce the bug, I believe I made a mistake with the fix you reverted - I will double check that later on today on my personal computer.

File: webmacs/commands/webjump.py [hunk 1]
```diff
@@ -259,15 +266,19 @@ def get_url(prompt):
...
+    if webjump:
         if len(args) > 1:
             # we found a webjump, now look for a single completion
-            completions = [c for c in webjump.complete_fn(args[1])
-                           if c.startswith(args[1])]
+            try:
+                completions = [c for c in webjump.complete_fn(
+                    args[1]) if c.startswith(args[1])]
+            except Exception:
+                completions = []
```
Comment (parkouss) [inline webmacs/commands/webjump.py]: Could you add a line like logging.exception("An error occurred during completion:") in the except clause, so the exception won't be silent?

**Comment by parkouss** [comment]
Oh, I am able to reproduce the issue - try to paste the url:

https://github.com/parkouss/webmacs/pull/24

and you will only reach about:blank without the commit I did.

**Comment by plafrance** [comment]
It seems to be working now. For some reason, I never had tried an url with subdirectories... not good.

**Comment by plafrance** [comment]
Oups! I accidentaly merged my branch "zoom" into bug_fix. I'm not confident enougn in my "git-skills" to try and revert the merge.  If there is a problem with the zoom part, just close the MR and I'll do it over again. Very sorry about that.

**Pull request merged**